bucket/pentaho-aggregation-designer.json

```diff
@@ -0,0 +1,50 @@
+{
+    "homepage": "https://pentaho.sourceforge.io/",
+    "description": "The Pentaho Aggregation Designer simplifies the creation and deployment of aggregate tables that improve the performance of your Pentaho Analysis (Mondrian) OLAP cubes.",
+    "license": {
+        "identifier": "GPL-2.0-only",
+        "url": "https://github.com/pentaho/pentaho-aggdesigner/blob/master/LICENSE.txt"
+    },
+    "version": "8.3.0.0-371",
+    "url": [
+        "https://downloads.sourceforge.net/project/pentaho/Pentaho%208.3/client-tools/pad-ce-8.3.0.0-371.zip",
+        "https://gist.githubusercontent.com/edgardmessias/13b3ed8b356a7bc5bc113f7a1a8ede35/raw/pad.ico"
```

Comment:
Downloading it from your gist is really not good idea.

Reply:
I use gist because not found file with .ico format to use in shortcut.
Any ideas as an alternative?

Reply:
If you did not find it then, where did you get it?

Reply:
Converted: png -> ico
Original file: https://github.com/pentaho/pentaho-aggdesigner/blob/master/pentaho-aggdesigner-ui/src/main/resources/org/pentaho/aggdes/ui/resources/images/window_icon.png

Reply:
@Ash258, I have a idea, improve the [lib/shortcuts.ps1](https://github.com/lukesampson/scoop/blob/master/lib/shortcuts.ps1) to check icon file type. For non icon file, convert to icon, see https://www.powershellgallery.com/packages/RoughDraft/0.1/Content/ConvertTo-Icon.ps1

Reply:
My solution:
```js
// copy icon file to "bucket\\pentaho-aggregation-designer.ico" and use:
    "pre_install": [
        "if (!$url) { $url = \"$(manifest_path $app $bucket)\" }",
        "$icon = $url -replace '.json$', '.ico'",
        "Copy-Item \"$icon\" \"$dir\\icon.ico\""
    ],
```